Store.d.ts

```diff
@@ -15,7 +15,7 @@ declare module "orbit-db-store" {
         /** 
          * Contains all entries of this Store
          */
-        all: any[];
+        all: any[] | {};
```

Comment:
this should only be `any[]`

https://github.com/orbitdb/orbit-db-store/blob/6a450aca4ba7f5a041677f5cf25632f0170cf7b7/src/Store.js#L155

Reply:
It should be, but the IDE told me it could be `{}` too, for some reason. uh, right, the implementation should be changed instead

Reply:
I dont think the code block I linked needs to be changed, and im not sure how it could return an object.

Reply:
Yes it shouldn't be. If for some reason it returns an object it's the fault of orbit-db not its typing.